docs/pages/concepts/logical/credentials.md

```diff
@@ -7,4 +7,40 @@ abstract: Meshery uses one or more Credentials when authenticating to a managed
 language: en
 list: include
 ---
-Meshery uses one or more Credentials when authenticating to a managed or unmanaged Connection.
\ No newline at end of file
+Meshery uses one or more Credentials when authenticating to a managed or unmanaged Connection.
```

Comment:
Technically, this is zero or more, however, that phrasing is a little difficult to digest, so, if you can include the concept of Meshery potentially not using a credential... great.